scripts/prepare-release.sh

```diff
@@ -70,6 +70,8 @@ sed -E s",^([[:space:]]+)image:.+$,\1image: $container_image," -i *yaml.template
 
 # Patch Helm chart
 sed -e s"/appVersion:.*/appVersion: $release/" -i deployment/node-feature-discovery/Chart.yaml
+sed -e s"/pullPolicy: Always$/pullPolicy: IfNotPresent/" -i deployment/node-feature-discovery/values.yaml
+sed -e s"/k8s-staging-nfd\/node-feature-discovery$/k8s-staging-nfd\/node-feature-discovery:$release/" -i deployment/node-feature-discovery/values.yaml
```

Comment:
The production repo is `k8s.gcr.io/nfd/node-feature-discovery`. nit: you could use some other separator than slash in order to improve readability
```suggest
    -e s"!gcr.io/k8s-staging-nfd/node-feature-discovery!k8s.gcr.io/nfd/node-feature-discovery!" \
    -i deployment/node-feature-discovery/values.yaml
```

Reply:
Thanks for the tip